examples/Microsoft.DBforPostgreSQL_flexibleServers@2023-06-01-preview/main.tf

```diff
@@ -0,0 +1,79 @@
+terraform {
+  required_providers {
+    azapi = {
+      source = "Azure/azapi"
+    }
+    azurerm = {
+      source = "hashicorp/azurerm"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {
+  }
+}
+
+provider "azapi" {
+  skip_provider_registration = false
+}
+
+variable "resource_name" {
+  type    = string
+  default = "acctest0001"
+}
+
+variable "replica_resource_name" {
+  type    = string
+  default = "acctestreplica0001"
+}
+
+variable "location" {
+  type    = string
+  default = "eastus"
+}
+
+data "azurerm_client_config" "current" {
+}
+
+resource "azapi_resource" "resourceGroup" {
+  type     = "Microsoft.Resources/resourceGroups@2020-06-01"
+  name     = var.resource_name
+  location = var.location
+}
+
+resource "azapi_resource" "flexibleServer" {
+  type      = "Microsoft.DBforPostgreSQL/flexibleServers@2023-06-01-preview"
+  parent_id = azapi_resource.resourceGroup.id
+  name      = var.resource_name
+  location  = var.location
+  body = {
+    identity = {
```

Comment:
Remove the identity block if it's none.

Reply:
It's required while provisioning the resource.